components/governance/org.wso2.carbon.governance.taxonomy.ui/src/main/resources/web/taxonomy/source_taxonomy.jsp

```diff
@@ -0,0 +1,173 @@
+<!--
+ ~ Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ ~
+ ~ WSO2 Inc. licenses this file to you under the Apache License,
+ ~ Version 2.0 (the "License"); you may not use this file except
+ ~ in compliance with the License.
+ ~ You may obtain a copy of the License at
+ ~
+ ~    http://www.apache.org/licenses/LICENSE-2.0
+ ~
+ ~ Unless required by applicable law or agreed to in writing,
+ ~ software distributed under the License is distributed on an
+ ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ ~ KIND, either express or implied.  See the License for the
+ ~ specific language governing permissions and limitations
+ ~ under the License.
+ -->
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<%@ taglib prefix="fmt" uri="http://java.sun.com/jsp/jstl/fmt" %>
+<%@ taglib prefix="carbon" uri="http://wso2.org/projects/carbon/taglibs/carbontags.jar" %>
+<%@ page import="org.wso2.carbon.utils.ServerConstants" %>
+<%@ page import="org.wso2.carbon.ui.CarbonUIMessage" %>
+<%@ page import="org.wso2.carbon.governance.taxonomy.ui.clients.TaxonomyManagementClient" %>
+
+<link type="text/css" rel="stylesheet" href="css/menu.css"/>
+<link type="text/css" rel="stylesheet" href="css/style.css"/>
+
+<!--Yahoo includes for dom event handling-->
+<script src="../yui/build/yahoo-dom-event/yahoo-dom-event.js" type="text/javascript"></script>
+
+<!--EditArea javascript syntax hylighter -->
+<script language="javascript" type="text/javascript" src="../editarea/edit_area_full.js"></script>
+
+<script type="text/javascript" src="../ajax/js/prototype.js"></script>
+<carbon:jsi18n
+		resourceBundle="org.wso2.carbon.governance.taxonomy.ui.i18n.JSResources"
+		request="<%=request%>" namespace="org.wso2.carbon.governance.taxonomy.ui"/>
+<script type="text/javascript" src="js/taxonomy.js"></script>
+<%
+    String cookie = (String) session.getAttribute(ServerConstants.ADMIN_SERVICE_COOKIE);
+    String temp = "";
+    boolean isNew = true;
+
+    boolean viewMode = false;
+    try{
```

Comment:
Thank you Sameera for mentioning it.
This has been fixed with https://github.com/wso2/carbon-governance/pull/253